WD-Firmware/rtos/rtosal/loc_inc/rtosal.h

```diff
@@ -45,6 +45,11 @@
 */
 void rtosalContextSwitchIndicationSet(void);
```

Comment:
change file name to rtosal_util.h

Reply:
done https://github.com/westerndigitalcorporation/riscv-fw-infrastructure/pull/12/commits/f1d341104d2182f0324edf87bb22f94fa83dabc7